### ISSUE TITLE
deprecated error in lspkind

### ISSUE THREAD

**Issue by desdic** [onsails/lspkind-nvim#39]
neovim 0.6

Error detected while processing /home/kgn/src/private/neovim/init.lua:
E5113: Error while calling lua chunk: ...site/pack/packer/start/lspkind-nvim/lua/lspkind/init.lua:123: Vim(echoerr):DEPRECATED replaced by mode option.
stack traceback:
        [C]: in function 'nvim_command'
        ...site/pack/packer/start/lspkind-nvim/lua/lspkind/init.lua:123: in function 'init'
        ...site/pack/packer/start/lspkind-nvim/lua/lspkind/init.lua:163: in function 'cmp_format'
        /home/kgn/.config/nvim/lua/config/cmp.lua:47: in main chunk
        [C]: in function 'require'
        /home/kgn/.config/nvim/init.lua:7: in main chunk

just updated

**Comment by WIttyJudge** [comment]
After updates.

Error detected while processing /home/wittyjudge/dotfiles/.config/nvim/init.lua:
E5113: Error while calling lua chunk: ...site/pack/packer/start/lspkind-nvim/lua/lspkind/init.lua:122: attempt to index local 'opts' (a nil value)
stack traceback:
        ...site/pack/packer/start/lspkind-nvim/lua/lspkind/init.lua:122: in function 'init'
        ...tyjudge/.config/nvim/lua/plugins-config/lspkind-nvim.lua:36: in main chunk
        [C]: in function 'require'
        /home/wittyjudge/.config/nvim/lua/plugins-config/init.lua:53: in main chunk
        [C]: in function 'require'
        /home/wittyjudge/.config/nvim/init.lua:39: in main chunk
Press ENTER or type command to continue

**Comment by Taloud** [comment]
> After updates.
> 
> Error detected while processing /home/wittyjudge/dotfiles/.config/nvim/init.lua: E5113: Error while calling lua chunk: ...site/pack/packer/start/lspkind-nvim/lua/lspkind/init.lua:122: attempt to index local 'opts' (a nil value) stack traceback: ...site/pack/packer/start/lspkind-nvim/lua/lspkind/init.lua:122: in function 'init' ...tyjudge/.config/nvim/lua/plugins-config/lspkind-nvim.lua:36: in main chunk [C]: in function 'require' /home/wittyjudge/.config/nvim/lua/plugins-config/init.lua:53: in main chunk [C]: in function 'require' /home/wittyjudge/.config/nvim/init.lua:39: in main chunk Press ENTER or type command to continue

Got same error about  `attempt to index local 'opts' ` with last version. had to disable lspkind for now ;s

**Comment by zeitchef** [comment]
For the moment, removing the option `with_text` from your config will suppress the error. Would have been nice to have a bit more notice on the deprecation, but it is what it is.

**Comment by romariolopezc** [comment]
@zeitchef I'm still having the error
My config:
```  formatting = {
    format = lspkind.cmp_format {
      mode = 'symbol',
      menu = {
        buffer = "[buf]",
        nvim_lsp = "[LSP]",
        ultisnips = "[snip]",
      },
    },
  },
```

Did I miss something?

**Comment by romariolopezc** [comment]
I fixed by adding `mode` in `lspkind.init({ mode = "symbol_text" })`

**Comment by demizer** [comment]
Causing the following error at startup:
```
Error detected while processing /data/dotfiles/nvim/plugin/nvim-cmp.lua:                                                                                                                            
E5113: Error while calling lua chunk: ...m/site/pack/paqs/start/lspkind-nvim/lua/lspkind/init.lua:122: attempt to index local 'opts' (a nil value)                                                  
stack traceback:                                                                                                                                                                                    
        ...m/site/pack/paqs/start/lspkind-nvim/lua/lspkind/init.lua:122: in function 'init'                                                                                                         
        /home/jesusa/.config/nvim/plugin/nvim-cmp.lua:5: in main chunk                                                                                                                              
```

**Comment by onsails** [comment]
Should be fixed by #40, thanks for spotting this.
